### PR TITLE
Add person client and person service

### PR DESCRIPTION
**Pull request by SandGrainOne**

### CODE REVIEW

File: src/Altinn.App.PlatformServices/Implementation/PersonClient.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            request.Headers.Add("X-Ai-LastName", lastName);
+
+            var response = await _httpClient.SendAsync(request, HttpCompletionOption.ResponseContentRead, ct);
+
+            return await ReadResponse(response, ct);
+        }
+
+        private void AddAuthHeaders(HttpRequestMessage request)
+        {
+            Application application = _appResources.GetApplication();
```
Comment (acn-sbuad) [inline src/Altinn.App.PlatformServices/Implementation/PersonClient.cs:87]: This is functionality we need for almost all of our clients. Is there a way to consolidate this, with the logic we have [here ](https://github.com/Altinn/app-template-dotnet/blob/main/src/Altinn.App.PlatformServices/Extensions/HttpClientExtension.cs#L21) Or at least make this method more widely available to reuse in our other clients as well?

Comment (SandGrainOne) [reply src/Altinn.App.PlatformServices/Implementation/PersonClient.cs:87]: I agree that there is a potential for a more seamless reuse here, but I'm not sure how yet. The code is currently using 3 different services and I don't like sending objects like that around as parameters.

Comment (SandGrainOne) [reply src/Altinn.App.PlatformServices/Implementation/PersonClient.cs:87]: I can split the method in two. One method to obtain the values and another one to set the headers. The last one could more easily be reused, but it doesn't change a whole lot. The goal should be to remove all of it and that requires more thought.

**Review by acn-sbuad — APPROVED** [review]
🎈

**Pull request merged**